WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -920,6 +934,28 @@ extension ProductsViewController: UITableViewDelegate {
     func scrollViewDidScroll(_ scrollView: UIScrollView) {
         hiddenScrollView.updateFromScrollViewDidScrollEventForLargeTitleWorkaround(scrollView)
     }
+
+    /// Provide an implementation to show cell swipe actions. Return `nil` to provide no action.
+    ///
+    func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        let product = resultsController.object(at: indexPath)
+        guard ServiceLocator.stores.sessionManager.defaultSite?.isPublic == true,
```

Comment:
Nit: I would suggest trying to move this to the view model just to decrease the number of lines in this already large view controller. 😄

Reply:
I think the move would make more sense when we can do a refactor to remove the `resultsController` to the view model. For now, moving only 1 line that's not specific to products isn't beneficial to me 😅. I'll keep this change for now if you don't mind.